data/installed-tests/fwupdmgr-p2p.sh

```diff
@@ -10,7 +10,7 @@ if [ "$(id -u)" -ne 0 ]; then exit 0; fi
 echo "Starting P2P daemon..."
 export FWUPD_DBUS_SOCKET="/var/run/fwupd.sock"
```

Comment:
If you're processing the file anyway shouldn't this be swapped out by configuration variable too?

Reply:
ohh hmm. Do we have any config for the socket location now? In the common code we have:

	/* /run/lock */
	case FU_PATH_KIND_LOCKDIR:
		return g_strdup("/run/lock");

Reply:
I think specifically systems directory to use for configuration (/run) can it be discovered using systemd pkgconfig?

Reply:
The twitterati seems to think that we should just hardcode /run now. I've made that change as a followup.